WebStoreApplication/Models/CoroNacessitiesDBContext.cs

```diff
@@ -7,7 +7,7 @@ public static class CoroNacessitiesDBContext
         private static SqlConnection connection;
         static CoroNacessitiesDBContext()
         {
-            connection = new SqlConnection ("Data Source=(LocalDb)\\LocalDBDemo;Database=CoroNacessitiesDB;Integrated Security=SSPI;");
+            connection = new SqlConnection ("Data Source=KEANUT;Database=CoroNacessitiesDB;Integrated Security=SSPI;");
```

Comment:
I think this needs to be changed back maybe to the default connection string?

Reply:
KEA-NUT

Reply:
;)